specification/resources/resource-manager/Microsoft.Authorization/stable/2023-04-01/policyAssignments.json

```diff
@@ -0,0 +1,1061 @@
+            "properties": {
+              "principalId": {
+                "readOnly": true,
+                "type": "string",
+                "description": "The principal id of user assigned identity."
+              },
+              "clientId": {
+                "readOnly": true,
+                "type": "string",
+                "description": "The client id of user assigned identity."
+              }
+            }
+          },
+          "description": "The user identity associated with the policy. The user identity dictionary key references will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'."
+        }
+      },
+      "description": "Identity for the resource.  Policy assignments support a maximum of one identity.  That is either a system assigned identity or a single user assigned identity."
+    },
+    "PolicyAssignmentUpdateProperties": {
+      "type": "object",
+      "properties": {
+        "resourceSelectors": {
+          "type": "array",
+          "items": {
+            "$ref": "../../common/v1/types.json#/definitions/ResourceSelector"
+          },
+          "x-ms-identifiers": [],
+          "description": "The resource selector list to filter policies by resource properties."
+        },
+        "overrides": {
+          "type": "array",
+          "items": {
+            "$ref": "../../common/v1/types.json#/definitions/Override"
+          },
+          "x-ms-identifiers": [],
+          "description": "The policy property value override."
+        }
+      },
+      "description": "The policy assignment properties for Patch request."
+    },
+    "PolicyAssignmentUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyAssignmentUpdateProperties",
+          "description": "The policy assignment properties for Patch request."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the policy assignment. Only required when utilizing managed identity."
+        },
+        "identity": {
+          "$ref": "#/definitions/Identity",
+          "description": "The managed identity associated with the policy assignment."
+        }
+      },
+      "description": "The policy assignment for Patch request."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "The ID of the target subscription.",
+      "x-ms-parameter-location": "method"
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the operation.",
+      "x-ms-parameter-location": "method"
```

Comment:
same above and same issue in other services

Reply:
Just made the change to reference the common types everywhere for sub id and api version